framework/view/view_test.go

```diff
@@ -391,3 +391,43 @@ func TestSvelteImportFromNodeModule(t *testing.T) {
 	is.In(res.Body().String(), "<time datetime=\"2022-07-19 10:19:00\">Jul 19, 2022</time>")
 	is.NoErr(app.Close())
 }
+
+func TestSvelteImportFromSvelte(t *testing.T) {
+	is := is.New(t)
+	ctx := context.Background()
+	dir := t.TempDir()
+	td := testdir.New(dir)
+	td.NodeModules["svelte"] = versions.Svelte
+	td.NodeModules["svelte-time"] = "*"
+	td.NodeModules["livebud"] = "*"
+	td.Files["controller/controller.go"] = `
+		package controller
+		type Controller struct {}
+		func (c *Controller) Index() string { return "" }
+	`
+	td.Files["ui/Time.svelte"] = `
+		<h1>The Time</h1>
+	`
+	td.Files["view/index.svelte"] = `
+	 	<script>
+			import Time from "ui/Time.svelte"
+		</script>
+		<Time/>
+	`
+	// only works with "import Time from "../ui/Time.svelte" however this is doesnt work with views in nested directories
```

Comment:
Do you mean that `../ui/time/Time.svelte` wouldn't work?

Reply:
No, I mean that does work, but only in the root of the view folder and not in any nested folders within it

Reply:
In a nested directory, does the following not work `../../ui/time/Time.svelte`? It should work with any relative path.

Reply:
No, doesn't work. It just returns an error: could not resolve ../../ui/Time.svelte. I definitely have the Time.svelte file in the UI folder.

Reply:
Ah cool, can you update the PR to demonstrate that? I'll fix this ASAP :)

Reply:
I've just done that